packages/devtools_app/test/fixtures/riverpod_app/pubspec.yaml

```diff
@@ -9,7 +9,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_riverpod: ^2.0.0-dev.5
+  flutter_riverpod: 
+    path: /Users/adson.leal/dev/personal/riverpod/packages/flutter_riverpod # TODO: need to use an actual version
```

Comment:
@rrousselGit we need to release a version first, before merging this PR.

Reply:
That's not a problem. It'll happen later